consoles/sshVirtsh.pm

```diff
@@ -315,6 +332,26 @@ sub add_disk {
 sub define_and_start {
     my ($self) = @_;
 
+    my $remote_vmm = "";
+    if ($self->vmm_family eq 'vmware') {
+        my $libvirtauthfilename = '/var/lib/libvirt/images/libvirt.auth';
```

Comment:
Nope. `/var/lib/libvirt/images/` is just our favorite dumpster folder, see `$xmlfilename`. Suggestions?